gamemode/modules/inventory/sv_inventory.lua

```diff
@@ -239,6 +239,25 @@ function inv.getSelectedItem( ply )
     return itemData.item, itemData.count
 end
 
+local function addDurabilityFuncs( wep, instance, itemData )
+    function wep:LoseDurability()
```

Comment:
Is this function unused? It's not returned nor called so unless there is something about lua that I never knew about I feel like there is a problem here.

Reply:
Nvm my bad this is fine